packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -4266,7 +4266,8 @@ export default class Meeting extends StatelessWebexPlugin {
               correlation_id: this.correlationId,
               locus_id: this.locusUrl.split('/').pop(),
               reason: error.message,
-              stack: error.stack
+              stack: error.stack,
+              turnDiscoveryEnabled: true
```

Comment:
This is not right. We don't always do turn discovery - see inside `doTurnDiscovery()` function - it doesn't do it for at least 2 possible reasons being checked at the beginning - you're already familiar with that code :-). 
Also, thinking about it a bit more... TURN discovery might fail and then we proceed without a TURN server and I'm wondering what value we should put in metrics for that case... Maybe instead of 'turnDiscoveryEnabled' we need 2 different fields: `turnDiscoverySkippedReason` and `turnServerUsed`:
- `turnDiscoverySkippedReason` would be a string 'config' if TURN discovery is disabled in config, 'reachability' if some clusters were reachable or `undefined` if TURN discovery is not skipped
- `turnServerUsed` - would be true only if TURN discovery resolved with some turnServerInfo (so it wasn't undefined)

cc @arun3528

Reply:
yes it doesn't do turn discovery every time but then if that's not attempted then it has not failed as well. Skipping is not failure, does it make sense to have skipping in failure metric